presubmit-service/check_if_pr_can_cherry_pick.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+
+# The organization containing the repo to be checked against, e.g. angular
+ORG=$1
+# The repo to be checked against, e.g. material2
+REPO=$2
+# The PR number to retrieve the patch files for, e.g. 1234
+PR=$3;
+# The branch in the repo to check for cherry-picking onto, e.g. 6.1.x
+BRANCH=$4;
+# The fullname of the repository to be checked against, e.g. angular/material2
+FULL_REPO="$ORG/$REPO"
+# Whether the PR can be cherry-picked on to the targetted branch.
+RESULT=false;
+
+# If the needed tmp directories don't exist, create them.
+[ ! -d "/tmp/presubmit_service/$FULL_REPO/" ] && mkdir -p /tmp/presubmit_service/$FULL_REPO;
+
+# If the local_repo directory does not exist, create it.
+[ ! -d ".presubmit_service/local_repo" ] && mkdir -p .presubmit_service/local_repo;
+cd .presubmit_service/local_repo;
+
+# If the current directory, which should be repos is not a toplevel of a git repo
+# initialize a repo and create a dummy commit in to create a local remote with a
+# master branch.
```

Comment:
This comment explains what he code is doing, but it would also benefit from some elaboration on why it does this